pkg/cluster/runcommand.go

```diff
@@ -0,0 +1,28 @@
+package cluster
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-10-01/compute"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+)
+
+func (u *simpleUpgrader) RunCommand(ctx context.Context, oc *api.OpenShiftManagedCluster, scaleSetName, instanceId string, parameters compute.RunCommandInput) *api.PluginError {
+	return nil
+}
+
+func (u *simpleUpgrader) RestartDocker(ctx context.Context, oc *api.OpenShiftManagedCluster, scaleSetName, instanceId string) *api.PluginError {
+	params := compute.RunCommandInput{}
```

Comment:
Is this where you will define  the actual run command?  Did I miss where those commands were defined?

Reply:
yes this is where the actual command will be specified. as this is wip at this point, I didn't add those yet but you can see a proposed example usage in the previous comment thread.